lib/browser/element.js

```diff
@@ -140,7 +140,11 @@ exports.waitForElementNotExist = function waitForElementNotExist(
 };
 
 exports.clickOn = function clickOn(selector) {
-  return this.getElement(selector).click();
+  return this.getElement(selector)
+    .then(elements => {
+      return elements.length > 1 ? elements : [elements];
+    })
```

Comment:
instead `this.getElements(selector)` and then it's always an array?

in general I think i'd prefer:

* breaking change (since we're already in a major bump PR): `clickOn()` now asserts that the selector only matches one element
* new method: `clickOnAll()` that calls `this.getElements()` and clicks them all like this

Reply:
well okay - your suggestion is not a breaking change then because the `clickOn` behavior doesn't change to the current one

Reply:
sure, let's punt on the `clickOn()` change, if we don't have any other actual breaking changes

Reply:
Oh, we do have a breaking change for the gofer 5 bump, ok, let's go ahead and have `clickOn()` assert